plugins/importexport/crossref/classes/CrossRefExportDom.inc.php

```diff
@@ -328,14 +328,17 @@ function &_generateJournalArticleDom(&$doc, &$journal, &$issue, &$section, &$art
 		XMLCustomWriter::appendChild($journalArticleNode, $titlesNode);
 
 		/* AuthorList */
-		$contributorsNode =& XMLCustomWriter::createElement($doc, 'contributors');
-		$isFirst = true;
-		foreach ($article->getAuthors() as $author) {
-			$authorNode =& $this->_generateAuthorDom($doc, $author, $isFirst);
-			$isFirst = false;
-			XMLCustomWriter::appendChild($contributorsNode, $authorNode);
+		// Contributor list must not be empty
```

Comment:
Are you working with articles without authors? Could you describe the use case a little more?